pages/services/beta-dcos-monitoring/0.5.0-beta/operations/prometheus/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: layout.pug
+navigationTitle: Configuring Prometheus
+title: Configuring Prometheus
+menuWeight: 30
+excerpt: Configuring Prometheus
+render: mustache
+model: ../../data.yml
+---
+
+#include /services/include/beta-software-warning.tmpl
+
+This section walks you through how to configure the Prometheus bundled with the {{ model.techName }} service.
```

Comment:
This section explains how to configure the Prometheus bundled with the {{ model.techName }} service.